ui/packages/consul-peerings/app/components/consul/peer/search-bar/index.hbs

```diff
@@ -81,7 +81,7 @@ as |key value|}}
     {{#each
       (get
         (require '/models/peer'
-          path='schema'
```

Comment:
Oh forgot to mention, this missed a PR somehow for the filtering thing, I was gonna PR it separate, but it slipped in here in the copy, I'm happy to leave it in here, but let me know if you want me to PR separate.